[Bug] Build fail

### Version
1.9.0
### Reproduction link
http://the.build.is.failing.nothing.to.link/
### Operating System
MacOS
### Device
Macbook Pro 15" 2018
### Browser & Version
Chrome 89.0.4389.90 (Official Build) (x86_64)
### Steps to reproduce
1. Download the bundle
2. Unzip
3. Build:

```sh
$ npm run install:clean

> material-dashboard-pro-react@1.9.0 install:clean
> rm -rf node_modules/ && rm -rf package-lock.json && npm install && npm start

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: @babel/core@7.9.0
npm ERR! node_modules/@babel/core
npm ERR!   @babel/core@"7.9.0" from react-scripts@3.4.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@"3.4.1" from the root project
npm ERR!   @babel/core@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!   node_modules/@svgr/webpack
npm ERR!     @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!     node_modules/react-scripts
npm ERR!       react-scripts@"3.4.1" from the root project
npm ERR!   9 more (babel-jest, babel-loader, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @babel/core@"^7.13.0" from @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12
npm ERR! node_modules/@babel/preset-env/node_modules/@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining
npm ERR!   @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@"^7.13.12" from @babel/preset-env@7.13.12
npm ERR!   node_modules/@babel/preset-env
npm ERR!     @babel/preset-env@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!     node_modules/@svgr/webpack
npm ERR!       @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!       node_modules/react-scripts
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/ludovic.maitre/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ludovic.maitre/.npm/_logs/2021-03-24T14_30_35_696Z-debug.log
npm ERR! code 1
npm ERR! path /Users/ludovic.maitre/Workspaces/Skunk/material-dashboard-pro-react-v1.9.0
npm ERR! command failed
npm ERR! command sh -c rm -rf node_modules/ && rm -rf package-lock.json && npm install && npm start

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ludovic.maitre/.npm/_logs/2021-03-24T14_30_35_770Z-debug.log
lmaitre-mbp:material-dashboard-pro-react-v1.9.0 ludovic.maitre$ npm --version
7.6.3
```
### What is expected?
Build works.
### What is actually happening?
Build fail.

---
### Solution

### Additional comments
I have bought the full bundle (250euros) and expect to have things working out of the box, please let me know the fix or when you have fixed, thanks a lot for your help,

Best regards.
<!-- generated by creative-tim-issues. DO NOT REMOVE -->

Version
1.9.0

Operating System
Windows10 Pro

Browser & Version
Google Chrome Version 89.0.4389.114 (Official Build) (64-bit)

Steps to reproduce
Download Material Dashboard Pro React
Unzip
Build

Node Version:
14.16.0

Terminal:
```
npm i --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: @babel/core@7.9.0
npm ERR! node_modules/@babel/core
npm ERR!   @babel/core@"7.9.0" from react-scripts@3.4.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@"3.4.1" from the root project
npm ERR!   @babel/core@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!   node_modules/@svgr/webpack
npm ERR!     @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!     node_modules/react-scripts
npm ERR!       react-scripts@"3.4.1" from the root project
npm ERR!   9 more (babel-jest, babel-loader, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @babel/core@"^7.13.0" from @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12
npm ERR! node_modules/@babel/preset-env/node_modules/@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining
npm ERR!   @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@"^7.13.12" from @babel/preset-env@7.13.12
npm ERR!   node_modules/@babel/preset-env
npm ERR!     @babel/preset-env@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!     node_modules/@svgr/webpack
npm ERR!       @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!       node_modules/react-scripts
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
What is expected?
Build works.

What is actually happening?
Build fail.

Additional comments

I am experiencing the same issue OP did. I've also done changing my node version to a Node LTS version  but the issue still persist. Please let me know the fix or if the issue has been fixed. Thanks in advance!

P.S, I don't know if it's proper to follow up a comment in this thread but since its open and have the same error I still put it here anyway. Do remind if I need to open up a different thread for this. Thank you very much!

> Version
> 1.9.0
> 
> Operating System
> Windows10 Pro
> 
> Browser & Version
> Google Chrome Version 89.0.4389.114 (Official Build) (64-bit)
> 
> Steps to reproduce
> Download Material Dashboard Pro React
> Unzip
> Build
> 
> Node Version:
> 14.16.0
> 
> Terminal:
> 
> ```
> npm i --save
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE unable to resolve dependency tree
> npm ERR!
> npm ERR! Found: @babel/core@7.9.0
> npm ERR! node_modules/@babel/core
> npm ERR!   @babel/core@"7.9.0" from react-scripts@3.4.1
> npm ERR!   node_modules/react-scripts
> npm ERR!     react-scripts@"3.4.1" from the root project
> npm ERR!   @babel/core@"^7.4.5" from @svgr/webpack@4.3.3
> npm ERR!   node_modules/@svgr/webpack
> npm ERR!     @svgr/webpack@"4.3.3" from react-scripts@3.4.1
> npm ERR!     node_modules/react-scripts
> npm ERR!       react-scripts@"3.4.1" from the root project
> npm ERR!   9 more (babel-jest, babel-loader, ...)
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer @babel/core@"^7.13.0" from @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12
> npm ERR! node_modules/@babel/preset-env/node_modules/@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining
> npm ERR!   @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@"^7.13.12" from @babel/preset-env@7.13.12
> npm ERR!   node_modules/@babel/preset-env
> npm ERR!     @babel/preset-env@"^7.4.5" from @svgr/webpack@4.3.3
> npm ERR!     node_modules/@svgr/webpack
> npm ERR!       @svgr/webpack@"4.3.3" from react-scripts@3.4.1
> npm ERR!       node_modules/react-scripts
> npm ERR!
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
> ```
> 
> What is expected?
> Build works.
> 
> What is actually happening?
> Build fail.
> 
> Additional comments
> 
> I am experiencing the same issue OP did. I've also done changing my node version to a Node LTS version but the issue still persist. Please let me know the fix or if the issue has been fixed. Thanks in advance!

Operating System: Windows 10 Pro

Good Day! I was now able to resolve the issue in my case.  Aside from downloading and installing a LTS version of Node JS you can also use command: npm install --save --legacy-peer-deps 
to address the version conflict.  Hope it helps others who may encounter this issue. Thank you very much!

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,69 @@
 # Change Log
 
+## [1.10.0] 2021-05-10
+### Bug fixing
+- Changed Multi Level Collapse links and layouts to stop it opening when going to the /admin/buttons page
+- Change SidebarWrapper from class to function component
+- Change Sidebar from class to function component
+  - Also, made the Multi Level Collapse stop opening when navigating to /admin/buttons
+  - Also, made the collapses close when opening a new one
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/253
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/252
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/251
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/243
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/232
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/229
+- https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues/228
+- Change Step1, Step2 and Step3 of Wizard component from class components to functional ones
+  - NOTE: The Wizard still accepts class components as well
+  - NOTE: The Wizard is still a class component. It could not be changed to functional component as of the moment :(. We will continue to find a solution for this component as well.
+### Major style changes
+### Deleted components
+### Added components
+### Deleted dependencies
+- `react-google-maps` (we've replaced this with simple Google Maps API)
+- `@types/googlemaps` (we've replaced this with simple Google Maps API)
+- `@types/markerclustererplus` (we've replaced this with simple Google Maps API)
+- `ajv` (this was only installed to stop some install warnings)
+- `history` (we'll use `BrowserRouter` from `react-router-dom` package instead)
+### Added dependencies
++ `@babel/core@7.14.0 `(to stop warnings)
+### Updated dependencies
+```
+@material-ui/core             4.9.14   →   4.11.4
+@material-ui/icons             4.9.1   →   4.11.2
+classnames                     2.2.6   →    2.3.1
+match-sorter                   4.1.0   →    6.3.0
+moment                        2.26.0   →   2.29.1
+node-sass                     4.14.1   →    5.0.0
+nouislider                    14.5.0   →   15.0.0
+perfect-scrollbar              1.5.0   →    1.5.1
+react                        16.13.1   →   17.0.2
+react-big-calendar            0.24.6   →   0.33.2
+react-bootstrap-sweetalert     5.1.9   →    5.2.0
+react-chartist                0.14.3   →   0.14.4
+react-datetime                2.16.3   →    3.0.4
+react-dom                    16.13.1   →   17.0.2
+react-scripts                  3.4.1   →    4.0.3
+react-table                    7.1.0   →    7.7.0
+eslint-config-prettier        6.11.0   →    8.3.0
+eslint-plugin-prettier         3.1.3   →    3.4.0
+gulp-append-prepend            1.0.8   →    1.0.9
+prettier                       2.0.5   →    2.2.1
+typescript                     3.9.3   →    4.2.4
+```
+### Warning
+_In the next version, we'll change all the ReactTable components to JSS based components, the same these tables were created: https://demos.creative-tim.com/material-dashboard-pro-react/#/admin/extended-tables or these https://demos.creative-tim.com/material-dashboard-pro-react/#/documentation/table_
+_While in development some of the plugins that were used for this product will throw some warnings - note, this only happens in development, the UI or the functionality of the product is not affected, also, if the issues will persist in React 17, we'll drop usage of those plugins, and replace them with other ones._
+_Warnings might appear while doing an npm install - they do not affect the UI or the functionality of the product, and they appear because of NodeJS and not from the product itself._
+```
+npm WARN react-datetime@3.0.4 requires a peer of react@^16.5.0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-event-listener@0.6.6 requires a peer of react@^16.3.0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-swipeable-views@0.13.9 requires a peer of react@^15.3.0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-tagsinput@3.19.0 requires a peer of react@^16.0.0 || ^15.0.0 || ^0.14.0 but none is installed. You must install peer dependencies yourself.
+```
+_You will also have the following message: `found 1 low severity vulnerability`. This comes from `gulp`, and will be fixed in the next version. NOTE: the product works as expected with these `vulnerabilities`._
+
 ## [1.9.0] 27.05.2020
 ### Bug fixing
 - Add `React.useEffect` inside `src/views/Pages/LoginPage.js` and `src/views/Pages/LockScreenPage.js` so that the `setTimeout` will not throw errors on route change

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 
 
-![version](https://img.shields.io/badge/version-1.9.0-blue.svg) [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/ct-material-dashboard-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues?q=is%3Aopen+is%3Aissue) [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/ct-material-dashboard-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-dashboard-pro/issues-react?q=is%3Aissue+is%3Aclosed) [![Join the chat at https://gitter.im/NIT-dgp/General](https://badges.gitter.im/NIT-dgp/General.svg)](https://gitter.im/creative-tim/material-dashboard) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
+![version](https://img.shields.io/badge/version-1.10.0-blue.svg) [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/ct-material-dashboard-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-dashboard-pro-react/issues?q=is%3Aopen+is%3Aissue) [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/ct-material-dashboard-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-dashboard-pro/issues-react?q=is%3Aissue+is%3Aclosed) [![Join the chat at https://gitter.im/NIT-dgp/General](https://badges.gitter.im/NIT-dgp/General.svg)](https://gitter.im/creative-tim/material-dashboard) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
 
 ![Product Gif](./material-dashboard-pro-react.gif)
 
@@ -276,7 +276,7 @@ If you have questions or need help integrating the product please [contact us](h
 
 ## Licensing
 
-- Copyright 2020 Creative Tim (https://www.creative-tim.com)
+- Copyright 2021 Creative Tim (https://www.creative-tim.com)
 - Creative Tim [license](https://www.creative-tim.com/license)
 
 ## Useful Links

